src/workflows-app/RunDetails.test.js

```diff
@@ -257,13 +257,20 @@ describe('BaseRunDetails - render smoke test', () => {
     const cellsFromDataRow2 = within(task2Row).getAllByRole('cell');
     within(cellsFromDataRow2[7]).getByText(/In Progress -/); // Can't accurately calculate cost for an 'In Progress' task in a test
 
+    // Running task with no start time should be fetching cost data
     const task3Row = tableRows[2];
     const cellsFromDataRow3 = within(task3Row).getAllByRole('cell');
-    within(cellsFromDataRow3[7]).getByText('-');
+    within(cellsFromDataRow3[7]).getByText('Fetching cost information');
```

Comment:
What if the workflow ran before we were saving cost data? Will it say "fetching cost information" forever?

Reply:
It shouldn't. In `noCostData` we check if `taskStartTime` is present. If not, the function returns true in conjunction with `executionStatus !== 'Running'` so a dash should be displayed as opposed to `Fetching cost information`